### ISSUE TITLE
Automate installation process through dpkg (.deb package)

### ISSUE THREAD

**Issue by nuxwin** [pydio/pydio-core#1309]
Good evening,

**The idea**

Right now, you're providing a set of package for Debian and derived distributions but the installation process is still left to the administrator. Hence, the administrator must create the database manually, fill the setup form through a browser and trigger installation... While this requirement is fine for standalone installations, that is not the case when Pydio is provided as part of a tool suite.

I would create an optional **pydio-db** package that would be able to initialize the Pydio database itself. This could be achieved using the **dbconfig-common** package that is provided for that exact purpose. We could provision the required data  through debconf during the package preconfiguration phase. What do you think about this idea? I would be really happy to help on that part.

**Why such a package?**

Currently, we provide Pydio instances through a composer package for our control panel. The installation is currently static (the composer package holds a preinstalled version of pydio) with the administrator account disabled by default. We think that it could be more easy for us to use your Debian packages.

**BTW:** Sorry for posting there. There is no dedicated repository for the **pydio-core** source package that is, by the way, not public.

Thank you.

**Comment by nuxwin** [comment]
Furthermore, asking users to process database update themself by executing some SQL statement manually when they use Pydio packages is not really good. The **dbconfig-common** package take care of database updates as well ;)

@cdujeu  Your thinking about this?

**Comment by cdujeu** [comment]
Definitely!!! The current upgrade process is our nightmare :) i'll come back to you on that!
